### DISCUSSION TITLE
Hierarchy in sidebar

### DISCUSSION THREAD

**Discussion by ShovalMizrahi** [samuelarbibe/dnd-timeline#27] [Q&A] [answered]
Does the library support hierarchy of rows in the sidebar?

**Comment by samuelarbibe** [comment, marked as answer]
Hey!
The library isn't responsible for rendering - so you can basically do anything you want (As long as you have rows and items of some shape, of course).

Luckily, I once played around with "nested" rows, here is an example you can take inspiration from:

https://github.com/samuelarbibe/dnd-timeline/blob/grouped/examples/grouped/src/Timeline.tsx

And here is a live example:

https://dnd-timeline-grouped.vercel.app/

Good luck!

**Comment by ShovalMizrahi** [comment]
thank you for your response.
what I meant was something like this:
[image: image.png]
but unfortunately I couldn't quite make it work

‫בתאריך יום ו׳, 10 במאי 2024 ב-18:32 מאת ‪Samuel Arbibe‬‏ <‪
***@***.***‬‏>:‬
